go-controller/pkg/config/config.go

```diff
@@ -130,8 +130,22 @@ var (
 
 	// Gateway holds node gateway-related parsed config file parameters and command-line overrides
 	Gateway = GatewayConfig{
-		V4JoinSubnet: "100.64.0.0/16",
-		V6JoinSubnet: "fd98::/64",
+		V4JoinSubnet:       "100.64.0.0/16",
+		V6JoinSubnet:       "fd98::/64",
+		V4MasqueradeSubnet: "169.254.169.0/29",
+		V6MasqueradeSubnet: "fd69::/125",
+		MasqueradeIPs: MasqueradeIPsConfig{
+			V4OVNMasqueradeIP:               net.ParseIP("169.254.169.1"),
+			V6OVNMasqueradeIP:               net.ParseIP("`fd69::1`"),
```

Comment:
why the backticks?

Reply:
must be some vscode formatting error, I corrected it